Makefile

```diff
@@ -84,8 +85,14 @@ else
 TESTCMD = -run $(RUNTEST)
 endif
 
+test_skip_kube: SKIP_KUBE=1
+test_skip_kube: test
+
 test:
-	@go test ./... -p 1 -parallel=16 $(TESTCMD) -count=1
+	@export PATH=$(PATH):$(PWD); \
+		export SKIP_KUBE=$(SKIP_KUBE); \
+		echo "Testing with receptor binary from: `which receptor 2>/dev/null`"; \
```

Comment:
Not all environments will have `which`, and this hides the error so it will be hard to tell what's going on without coming to look at this code.